extensions/pkg/controller/backupbucket/controller.go

```diff
@@ -42,16 +42,17 @@ type AddArgs struct {
 	// The options.Reconciler is always overridden with a reconciler created from the
 	// given actuator.
 	ControllerOptions controller.Options
-	// Predicates are the predicates to use.
-	// If unset, GenerationChangedPredicate will be used.
-	Predicates []predicate.Predicate
```

Comment:
Why is this removed from the `AddArgs`? I think the idea was that the caller can provide custom provides here.

Reply:
Added it again. Now during extension registration, the OperationAnnotation flag has to be handed over twice, but thats the price.